exercises/reverse-string/example.go

```diff
@@ -0,0 +1,10 @@
+package reverse
+
+// String reverses given string
+func String(s string) string {
```

Comment:
would have preferred
```
func Reverse(s string) string {
```
for naming sake.  But since package name is reverse, the call is reverse.String(x) which does read well in itself, perhaps better than reverse.Reverse(x).  Optional.

Reply:
I preferred `String` due package naming and you picked this. Let's wait for @tleen to break even case :wink:

Reply:
It gets a little awkward with reverse.Reverse and since you could technically reverse an int or something using the type name as the function name seems ok to me.

Reply:
Thanks, that was similar to the flow of my logic at first so merging.